lower.ts

```diff
@@ -267,7 +267,7 @@ function flattenExprToExpr(e : AST.Expr<[Type, SourceLocation]>, env : GlobalEnv
       const classdata = env.classes.get(e.name);
       const fields = [...classdata.entries()];
       const newName = generateName("newObj");
-      const alloc : IR.Expr<[Type, SourceLocation]> = { tag: "alloc", amount: { tag: "wasmint", value: fields.length } };
+      const alloc : IR.Expr<[Type, SourceLocation]> = { tag: "alloc", amount: { tag: "wasmint", value: BigInt(fields.length) } };
```

Comment:
It's surprising to me that this change is made to the argument to alloc, but there's no change to the _implementation_ of alloc. What happens if alloc gets an actual big number? Aren't bignums heap allocated, won't that affect how alloc works?